docker_deep_dive_part_1/Deploy_Wordpress_With_Multiple_Container.md

```diff
@@ -0,0 +1,66 @@
+## Deploy Wordpress With Multiple Container 
```

Comment:
At some places you are using MYSQL and image you are using is maria db. Please make that consistent to avoid confusion.